### ISSUE TITLE
I DONT KNOW WHY ITS NOT RUNING 

### ISSUE THREAD

**Issue by Hamadalkabbi2** [MrChromebox/scripts#372]
I did the step-by-step guide in Coolstar but it keeps giving error 
![photo_5390839785557453879_y](https://github.com/MrChromebox/scripts/assets/131576352/4655e75c-5a90-4dec-9494-77ed99d0be07)

**Comment by MrChromebox** [comment]
SMH. Is it that hard to follow the directions on my site? I don't know why you're using a root shell, the instructions explicitly say not to. https://mrchromebox.tech/#fwscript
